scintillometry/fourier/base.py

```diff
@@ -0,0 +1,273 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import numpy as np
+import operator
+from astropy.utils import lazyproperty
+
+
+class FFTMakerBase(object):
```

Comment:
If we give this class a metaclass that tracks subclasses (or use the new `__init_subclass__` - maybe it is fine to requite python >=3.6...), then there is no need to keep the hand-crafted list in `__init__.py`.